frontend/src/lib/k8s/configMap.ts

```diff
@@ -1,12 +1,21 @@
 import { apiFactoryWithNamespace } from './apiProxy';
-import { KubeObjectInterface, makeKubeObject, StringDict } from './cluster';
+import { BaseKubeObject, KubeObjectInterface, makeKubeObject, StringDict } from './cluster';
 
 export interface KubeConfigMap extends KubeObjectInterface {
   data: StringDict;
 }
 
-class ConfigMap extends makeKubeObject<KubeConfigMap>('configMap') {
+class ConfigMap extends makeKubeObject<KubeConfigMap>('configMap') implements BaseKubeObject {
```

Comment:
This shouldn't be needed when you add the getBaseObject to the KubeConfig class.

Reply:
Removed